tests/testrom.c

```diff
@@ -185,6 +194,7 @@ static const struct Testsuite
 	TEST_FUNC(test_irq_reentrancy,       230, TEST_FLAGS_RESET_COUNT),
 	TEST_FUNC(test_dfs_read,            1104, TEST_FLAGS_IO),
 	TEST_FUNC(test_cache_invalidate,    1763, TEST_FLAGS_NONE),
+	TEST_FUNC(test_debug_sdfs,             0, TEST_FLAGS_NO_BENCHMARK),
```

Comment:
The test code is for correctness; we can't benchmark SD accesses because it will be dependent on the actual speed of the physical SD being inserted and also the status of the FAT partition in it.